### ISSUE TITLE
How to set timeouts?

### ISSUE THREAD

**Issue by peeeto** [Prominence/openweathermap-java-api#31]
Hallo,

how could I please set timeouts in the client?
```
@Bean
    public OpenWeatherMapClient openWeatherClient() {
        OpenWeatherMapClient openWeatherClient = new OpenWeatherMapClient(KEY);
        openWeatherClient.setReadTimeout(1000L);//<---this is missing
        openWeatherClient.setConnectTimeout(1000L);//<---this is missing
        return openWeatherClient;
    }
```

Thanks.

**Comment by Prominence** [comment]
Hello!
Currently, there is no such option implemented.
But I could add it to my TODO list, when I'll find some time for this project.

Also, you can offer your PR if you'll implement it by yourself.

**Comment by Prominence** [comment]
Implemented in **2.3.0** release.
